pkg/api/hosttwin.go

```diff
@@ -0,0 +1,62 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/Iotic-Labs/iotics-identity-go/pkg/advancedapi"
+	"github.com/Iotic-Labs/iotics-identity-go/pkg/register"
+)
+
+// DelegateControlByPrivateExponent registers a twin identity with a control delegate to agent
+func DelegateControlByPrivateExponent(resolverClient register.ResolverClient, twinIssuer *register.Issuer, twinPrivateExponent string, agentID register.RegisteredIdentity, delegationName string) error {
+	twinKeypair, err := advancedapi.GetKeyPairFromPrivateExponentHex(twinPrivateExponent)
+	if err != nil {
+		return err
+	}
+
+	twinID := register.NewRegisteredIdentity(twinKeypair, twinIssuer)
+
+	return TwinDelegatesControlToAgent(resolverClient, twinID, agentID, delegationName)
+}
+
+// GetOwnershipOfTwinByPrivateExponent Get Ownership of a twin using the private exponent of the twin.
+func GetOwnershipOfTwinByPrivateExponent(resolverClient register.ResolverClient, twinIssuer *register.Issuer, twinPrivateExponent string, newOwnerID register.RegisteredIdentity, newOwnerKeyName string) error {
```

Comment:
Is this *Set* (or *Add*) instead of *Get*, judging by what this does?

Reply:
Oh ok, "Get ownership" means... "Take ownership"? But I think "Get" might make people think of getter methods. Would "Take" or "Add" (is it replace) work better maybe?

Reply:
I've changed them from Get to Take and added Hex on the end.